cs-config/install.sh

```diff
@@ -2,5 +2,15 @@
 
 git clone https://github.com/PSLmodels/OG-USA
```

Comment:
This saves about 0.75 GB which makes building and pushing OG-USA much faster.

```suggestion
git clone -b master --depth 1 https://github.com/PSLmodels/OG-USA
```